agent/core/src/main/java/org/jolokia/config/ConfigKey.java

```diff
@@ -222,6 +223,12 @@
      */
     AUTH_MODE("authMode", true, false, "basic"),
 
+    /**
+     * If MultiAuthenticator is used, this config item explains how to combine multiple authenticators
+     * Supported values: "any" at least one authenticator must match, "every" or "all" all authenticators must match
+     */
+    AUTH_PASS("authPass",true, false, "any"),
```

Comment:
can we ommit 'every' and use only 'all' ? Doesn't make much sense to keep both and only increases the maintenance burden.

Reply:
Maybe we rename it to `authMatch` instead of `authPass` ? Not sure, sounds a better fit to me ...